.circleci/scripts/chrome-install

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Could you rename this script to use the `.sh` file extension, so that it'll be covered by ShellCheck?

Reply:
So that it's covered in CI I mean. The script doesn't have any ShellCheck violations at the moment - it looks quite well written!